gnat2goto/driver/tree_walk.adb

```diff
@@ -1779,6 +1826,11 @@ package body Tree_Walk is
          Do_Pragma_Assert_or_Assume (N_Orig, Block);
       elsif Pragma_Name (N_Orig) in Name_Annotate then
          null; -- ignore here. Rather look for those when we process a node.
+      elsif Pragma_Name (N_Orig) in Name_SPARK_Mode | Name_Global |
+      Name_Postcondition | Name_Refined_State | Name_Refined_Global |
+      Name_Precondition | Name_Loop_Invariant
+      then
+         null;
```

Comment:
Leave a comment that these are ignored because they're not implemented yet, unlike the previous one which is just implemented elsewhere.